ci/ci-test.sh

```diff
@@ -107,7 +107,7 @@ runTestSuite() {
 
   echo "running ginkgo test case with coverage ${coverageFile}"
 
-  if ! ginkgo -v -coverprofile="${coverageFile}" --label-filter="${labelFilter}" -covermode=atomic; then
+  if ! ginkgo -p -v -coverprofile="${coverageFile}" --label-filter="${labelFilter}" -covermode=atomic; then
```

Comment:
This runs the tests in parallel?
But it's still on the same cluster, is it better to split into different clusters?

Reply:
removed the `-p`